integration/kubernetes/vfio.sh

```diff
@@ -127,6 +127,8 @@ run_test() {
 	else
 		info "Success: found 2 network interfaces"
 	fi
+
+	sudo -E kubectl delete pod "${pod_name}"
```

Comment:
For symmetry, wouldn't it be nicer with a

```
kubectl delete -f "${SCRIPT_DIR}/runtimeclass_workloads/vfio.yaml"
```

?

Assuming maybe some day the manifest may need to add other resources.